new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/LoggingExporter.java

```diff
@@ -0,0 +1,474 @@
+    boolean deploymentCreated = assertDoesNotThrow(() -> Kubernetes.createDeployment(kibanaDeployment),
+        String.format("Create Kibana deployment failed with ApiException for %s in namespace $s",
+            kibanaName, namespace));
+    assertTrue(deploymentCreated,
+        String.format("Create Kibana deployment failed with ApiException for %s in %s namespace",
+            kibanaName, namespace));
+    logger.info("Checking if Kibana deployment is ready {0} completed in namespace {1}",
+        kibanaName, namespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for Kibana deployment {0} to be completed in namespace {1}"
+                + "(elapsed time {2}ms, remaining time {3}ms)",
+                kibanaName,
+                namespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(Deployment.isReady(kibanaName, labels, namespace));
+
+    // create Kibana service CR
+    logger.info("Create Kibana service CR for {0} in namespace {1}", kibanaName, namespace);
+    V1Service kibanaService = createKibanaServiceCr(params);
+
+    // create Kibana service
+    logger.info("Create Kibana service for {0} in namespace {1}", kibanaName, namespace);
+    boolean serviceCreated = assertDoesNotThrow(() -> Kubernetes.createService(kibanaService),
+        String.format("Create Kibana service failed with ApiException for %s in namespace %s",
+            kibanaName, namespace));
+    assertTrue(serviceCreated, String.format(
+        "Create Kibana service failed with ApiException for %s in namespace %s",
+            kibanaName, namespace));
+
+    // check that Kibana service exists in its namespace
+    logger.info("Checking that Kibana service {0} exists in namespace {1}",
+        kibanaName, namespace);
+    checkServiceExists(kibanaName, namespace);
+
+    return true;
+  }
+
+  /**
+   * Uninstall Elasticsearch.
+   *
+   * @param params the parameters to uninstall Elasticsearch
+   * @return true if the Elasticsearch is successfully uninstalled, false otherwise.
+   */
+  public static boolean uninstallElasticsearch(LoggingExporterParams params) {
+    String elasticsearchName = params.getElasticsearchName();
+    String namespace = params.getLoggingExporterNamespace();
+
+    // Delete Elasticsearch deployment
+    try {
+      Kubernetes.deleteDeployment(namespace, elasticsearchName);
+    } catch (Exception ex) {
+      logger.warning("Failed to delete deployment {0} ", elasticsearchName);
+    }
+
+    logger.info("Delete Service {0} in namespace {1}", elasticsearchName, namespace);
+    boolean serviceDeleted =
+        assertDoesNotThrow(() -> Kubernetes.deleteService(elasticsearchName, namespace),
+            String.format("Delete service failed with ApiException for %s in %s namespace ",
+              elasticsearchName, namespace));
+    assertTrue(serviceDeleted, String.format(
```

Comment:
change the message as if it failed with ApiException it is captured in the above line

Reply:
changed the line 195